protos/proto/aptos/transaction/v1/transaction.proto

```diff
@@ -59,6 +60,7 @@ message Transaction {
     GenesisTransaction genesis = 8;
     StateCheckpointTransaction state_checkpoint = 9;
     UserTransaction user = 10;
+    ValidatorTransaction validator = 21;
```

Comment:
should this be 11?

Reply:
@bowenyang007

Reply:
doesn't matter. leaving some space might be good 🤷

Reply:
11 works too.